src/main/java/thestonedturtle/lootlogger/ui/LootPanel.java

```diff
@@ -108,7 +108,7 @@ private void createPanel(final LootLog lootLog)
 			{
 				final Collection<UniqueItem> uniques = positionMap.get(position);
 
-				final UniqueItemPanel p = new UniqueItemPanel(uniques, this.itemManager);
+				final UniqueItemPanel p = new UniqueItemPanel(uniques, this.itemManager, this.config);
```

Comment:
Instead of passing the entire config object just pass the opacity value.

Grab the value outside of the loop as config operations are pretty costly.